wen-restart/src/wen_restart.rs

```diff
@@ -247,9 +258,10 @@ pub(crate) fn find_heaviest_fork(
     blockstore: Arc<Blockstore>,
     exit: Arc<AtomicBool>,
 ) -> Result<(Slot, Hash)> {
-    // Because everything else is stopped, it's okay to grab a big lock on bank_forks.
-    let my_bank_forks = bank_forks.read().unwrap();
-    let root_bank = my_bank_forks.root_bank().clone();
+    let root_bank;
+    {
+        root_bank = bank_forks.read().unwrap().root_bank().clone();
+    }
```

Comment:
nit: No need for the extra scope, can just define `root_bank` immediately right:
```rust
let root_bank = bank_forks.read().unwrap().root_bank();
```
Also, shouldn't need for the `.clone()` as `BankForks::root_bank()` already returned an `Arc<Bank>`

Reply:
Changed.

Reply:
ping

Reply:
Oops, fixed a previous location and missed this, changed.